cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfRobotComponent.cpp

```diff
@@ -55,3 +52,31 @@ void UUrdfRobotComponent::createChildComponents(UrdfLinkDesc* parent_link_desc,
         createChildComponents(child_link_desc, child_link_component);
     }
 }
+
+void UUrdfRobotComponent::applyKeyInputs()
```

Comment:
Move to the pawn level. The joint components should be publicly accessible to the pawn, so this method could be just as easily implemented at the pawn level. Why are only positive actions allowed?